### PR TITLE
fix(sdk): Fix alignment of ColorSelector in SDK Chart Settings

### PR DESCRIPTION
**Pull request by oisincoveney**
Fixes the ColorSelector in the `QuestionSettings` component of the SDK by converting the color selector's popover component to a Mantine popover. For whatever reason, the width of the ColorSelectorPopover reverts to a 1x32 (or however many colors there are) in the SDK, but with the Mantine popover it keeps its shape.

No tests and no styles have changed. The best way to test it is to make sure it is very much the same experience in the visualization settings for a question.

Before:

image

After:

image

### CODE REVIEW

**Comment by oisincoveney** [comment]
This change is part of the following stack:

- #51000 ◀

<sub>Change managed by [git-spice](https://abhinav.github.io/git-spice/).</sub>

File: frontend/src/metabase/core/components/ColorSelector/ColorSelector.tsx [hunk 1]
```diff
@@ -19,26 +19,29 @@ export interface ColorSelectorProps extends ColorSelectorAttributes {
...
   return (
-    <TippyPopoverWithTrigger
-      renderTrigger={({ onClick }) => (
-        <ColorPill {...props} ref={ref} color={value} onClick={onClick} />
-      )}
-      popoverContent={({ closePopover }) => (
+    <Popover opened={opened} onClose={close} position="bottom-start">
+      <Popover.Target>
+        <Center>
+          <ColorPill {...props} color={value} onClick={toggle} />
```
Comment (heypoom) [inline frontend/src/metabase/core/components/ColorSelector/ColorSelector.tsx:34]: Do we still need the ref here?

**Review by heypoom — APPROVED** [review]

File: frontend/src/metabase/core/components/ColorSelector/ColorSelector.tsx (hunk 1 above)
Comment (oisincoveney) [reply frontend/src/metabase/core/components/ColorSelector/ColorSelector.tsx:34]: I don't think so

**Review by kamilmielnik — APPROVED** [review]
:+1:

**Pull request merged**